internal/xds/translator/runner/runner.go

```diff
@@ -78,6 +78,10 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 				}
 
 				result, err := t.Translate(val)
+				if err != nil {
```

Comment:
you can't `return` here, this is an endless loop, one bad config should not cause the entire translator to exit

Reply:
sure, let me change it to move under the `else` directive.

Reply:
out of curiosity, @arkodg if we are not allowed to `return` this, then why this is allowed here? 
https://github.com/envoyproxy/gateway/blob/2ba50ea89e7a7342f3c44962715201b2d382b862/internal/provider/kubernetes/controller.go#L1077-L1093
Because I thought that process was just a callable function, that would be executed on every incoming data.

Reply:
that is returning from the function, not from the go routine

Reply:
sure, just a side note, I forgot to mention as well, that the patch that I work on now is actually running in my org staging environment, and it works well.